test/brsTypes/components/Font.test.js

```diff
@@ -0,0 +1,23 @@
+const brs = require("brs");
+const { Font } = brs.types;
+
+describe("Font", () => {
+    describe("stringification", () => {
+        it("inits a new Font component", () => {
+            let group = new Font();
+
+            expect(group.toString()).toEqual(
+                `<Component: roSGNode:Font> =
+{
+    change: <UNINITIALIZED>
+    focusable: false
+    focusedchild: invalid
+    id: 
+    uri: 
+    size: 1
+    fallbackglyph: 
+}`
```

Comment:
This is _just barely_ different than RBI — looks like `id` and `uri` default to empty strings!

```
<Component: roSGNode:Font> =
{
    fallbackGlyph: ""
    size: 1
    uri: "foo"
    change: <Component: roAssociativeArray>
    focusable: false
    focusedChild: <Component: roInvalid>
    id: ""
}
```

Would you mind setting defaults for those in `Font.ts` (and maybe `RoSGNode.ts` for the base)?

Reply:
After chatting out of band, we're going to solve this in https://github.com/sjbarag/brs/issues/416 !